src/gui/lazyfilelist.h

```diff
@@ -227,30 +234,30 @@ class LazyDirView {
                 }
             }
         }
-        void CopyFrom(const FILINFO &fno) {
-            size_t charsCopied = strlcpy(lfn, fno.fname, sizeof(lfn));
-            if (fno.altname[0] == 0) { // the lfn is identical to sfn
-                strlcpy(sfn, lfn, sizeof(sfn));
-            } else {
-                strlcpy(sfn, fno.altname, sizeof(sfn));
-            }
+        void CopyFrom(const dirent *fno) {
+            strlcpy(sfn, fno->d_name, sizeof(sfn));
+            strlcpy(lfn, fno->lfn, sizeof(lfn));
+            //            //@@TODO create function to get lfn from FATfs
```

Comment:
Whole file should be reviewed for FatFS mentions, as it should be POSIX now.

Also, please, pay attention to the coding style (extra spaces, double slashes).